api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -17,19 +17,13 @@
 
   DbUser createServiceAccountUser(String email);
 
-  DbUser createUser(
-      String givenName,
-      String familyName,
-      String email,
-      String contactEmail,
-      String currentPosition,
-      String organization,
-      String areaOfResearch);
+  // minimal version used by AuthInterceptor
+  DbUser createUser(String givenName, String familyName, String userName);
 
   DbUser createUser(
       String givenName,
       String familyName,
-      String email,
+      String userName,
```

Comment:
we must have missed this while standardizing on the new name

Reply:
Yes. Sometimes the refactoring is smart enough to do what you want with these, but if one of the names is a substring of other identifiers, all heck breaks loose. Safer to risk leaving some local identifiers behind like this.